src/ray/core_worker/common.h

```diff
@@ -177,6 +184,9 @@ struct ActorCreationOptions {
   const int max_pending_calls = -1;
   // The strategy about how to schedule this actor.
   rpc::SchedulingStrategy scheduling_strategy;
+  /// True if task events (worker::TaskEvent) from this creation task should be reported
+  /// default to true.
+  const bool task_tracing = true;
```

Comment:
Q: is the default value required here?

Is it possible to have one top level default value in core worker instead of having default value everywhere?